src/test/java/com/ays/AbstractRestControllerTest.java

```diff
@@ -52,6 +54,7 @@ public void initializeAuth() {
         this.tokenConfiguration = new AysTokenConfigurationParameter(parameterService);
         this.mockAdminUserToken = this.generate(new AdminUserEntityBuilder().build().getClaims());
         this.mockUserToken = this.generate(new UserEntityBuilder().build().getClaims());
+        this.mockSuperAdminToken = this.generate(new SuperAdminEntityBuilder().withValidFields().build().getClaims());
```

Comment:
Bu kısmı daha okunaklı olması için aşağıdaki gibi sıralayabilir miyiz?

```suggestion
        this.tokenConfiguration = new AysTokenConfigurationParameter(parameterService);
        this.mockSuperAdminToken = this.generate(new SuperAdminEntityBuilder().withValidFields().build().getClaims());
        this.mockAdminUserToken = this.generate(new AdminUserEntityBuilder().build().getClaims());
        this.mockUserToken = this.generate(new UserEntityBuilder().build().getClaims());
```

Reply:
Bu kısmı revise ettim